Projekty/Projekt1/Gruszkowska_Klaudia_Jamrozy_Bartosz/KM2.ipynb

```diff
@@ -0,0 +1,3476 @@
+{
```

Comment:
<p>Skoro klasy w przypadku klasyfikacji binarnej są tak bardzo niezbalansowane, lepiej byłoby we wnioskach powołać się na inną miarę jakości.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>